build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -56,22 +56,38 @@ function(iree_bytecode_module)
   if(DEFINED _RULE_TRANSLATE_TOOL)
     set(_TRANSLATE_TOOL ${_RULE_TRANSLATE_TOOL})
   else()
-    set(_TRANSLATE_TOOL "iree_tools_iree-translate")
+    set(_TRANSLATE_TOOL "iree-translate")
   endif()
 
-  # Resolve the executable binary path from the target name.
-  set(_TRANSLATE_TOOL_EXECUTABLE $<TARGET_FILE:${_TRANSLATE_TOOL}>)
+  if (CMAKE_CROSSCOMPILING)
```

Comment:
move this if into iree_get_host_executable_path and just call it everywhere

Reply:
Done.